client/eslint.config.js

```diff
@@ -0,0 +1,33 @@
+import js from '@eslint/js'
+import globals from 'globals'
+import reactHooks from 'eslint-plugin-react-hooks'
+import reactRefresh from 'eslint-plugin-react-refresh'
+
+export default [
```

Comment:
These are all bootstrap defaults; we can revisit them later. I'm guessing Typescript will be a topic of discussion!